pkg/commands/workload_get.go

```diff
@@ -114,6 +115,10 @@ func (opts *WorkloadGetOptions) Exec(ctx context.Context, c *cli.Config) error {
 		return nil
 	}
 
+	if cli.CommandFromContext(ctx).Flags().Changed(cli.StripDash(flags.NoColorFlagName)) {
```

Comment:
Can we set these NoColor flag to config in main ?

Reply:
Thanks for your suggestion, I already updated the PR